### ISSUE TITLE
zsys ignores a newly migrated user dataset that replaced the default installed dataset.

### ISSUE THREAD

**Issue by odror** [ubuntu/zsys#202]
**Describe the bug**
When migrating user dataset from an old system to the new one. It is not updated by zsys.

**To Reproduce**

use zfs send/receive to move the dataset. Then rename mount directory  or remove  the current dataset so that it will not conflict. verify that the mount directory in the received dataset is correct.

zsysctl save - v

should generate zsys snapshot.  It does not.

I get the following error.

INFO Requesting to save state for user "dror"     
INFO Couldn't find any association for user dataset rpool/USERDATA/dror_zo0whd 

**Expected behavior**
a zsys snapshot should be generated.

1. `ubuntu-bug zsys --save=/tmp/report`
2. Copy paste below `/tmp/report` content:
```
[report.txt](https://github.com/ubuntu/zsys/files/6191175/report.txt)

```

**Screenshots**
None
**Installed versions:**
 - OS: 5.11.0-11-generic #12-Ubuntu SMP Mon Mar 1 19:26:56 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux
- Zsysd running version: 0.5.7

**Additional context**
None

**Comment by odror** [comment]
I found a workaround for the issue.
I had to change the dateset property property **_com.ubuntu.zsys:bootfs-datasets_** to match the new computer.

**Comment by didrocks** [comment]
This is not a workaround, but the actual implementation :)

I would rephrase the bug to something like a feature request: "Please include a command to include new user datasets" (this would ensure the dataset is set in <pool>/USERDATA/ + proceed the same tagging as you did). Mind retitling it?
